tests/test_mock_controller.py

```diff
@@ -29,6 +29,7 @@
 
 from lsst.ts import salobj
 from lsst.ts import mtmount
+from lsst.ts import utils
```

Comment:
Same bit of nitpicking but wouldn't `from lsst.ts import salobj, mtmount, utils` make sense here?